docs/static/persistent-queues.asciidoc

```diff
@@ -91,6 +91,43 @@ first.
 See <<backpressure-persistent-queue>> for behavior when queue limits are
 reached.
 
+Calculating the queue size will vary depending on the use-case. As a general guiding
+principle, you should consider the following formula to size your Persistent Queue.
+
+-------
+Total Event Size per second = Input EPS * Original Event Size
+Total Received Per Hour = Total Event Size per Second * 3600s
+Tolerated Downtime = (Total Received Per Hour * Number of hours) * Multiplication Factor <1>
```

Comment:
```suggestion
Bytes Received Per Second = Incoming Events Per Second * Raw Event Byte Size
Bytes Received Per Hour = Bytes Received per Second * 3600s
Required Queue Capacity = (Bytes Received Per Hour * Tolerated Hours of Downtime) * Multiplication Factor <1>
```

Reply:
`Required Queue Capacity` can also be "queue.max_bytes" as that would be a "call to action" to set the configuration option.